subsys/bluetooth/mesh/lpn.c

```diff
@@ -912,7 +917,7 @@ static void lpn_timeout(struct k_work *work)
 		break;
 	case BT_MESH_LPN_RECV_DELAY:
 		k_work_reschedule(&lpn->timer,
-				  K_MSEC(lpn->adv_duration + SCAN_LATENCY + lpn->recv_win));
+			K_MSEC(SCAN_LATENCY + lpn->recv_win + RX_DELAY_CORRECTION));
```

Comment:
minor: you're unnecessarily (and incorrectly) changing the indentation/alignment of the line